pkg/identity/identity.go

```diff
@@ -140,6 +140,11 @@ func (id *Identity) IsWellKnown() bool {
 	return WellKnown.lookupByNumericIdentity(id.ID) != nil
 }
 
+// IsGlobal returns true if the identity is backed by a global identity.
+func (id *Identity) IsGlobal() bool {
+	return RequiresGlobalIdentity(id.Labels)
```

Comment:
Discussed offline that this kind of functionality could act upon `NumericIdentity` instead to simplify merge with #7538. Alternatively #7616 may provide a better alternative without needing to interact at this layer.

Reply:
Will look into rebasing against #7616.